main.ts

```diff
@@ -282,19 +310,29 @@ class BeautitabPluginSettingTab extends PluginSettingTab {
 			.setDesc(
 				`Which plugin should be utilized for search when clicking the middle of the screen button?`
 			)
-			.addDropdown((component) => {
-				SearchProviders.forEach((provider) => {
-					component.addOption(provider.value, provider.display);
-				});
-
-				component.setValue(this.plugin.settings.inlineSearchProvider);
+			.setClass("search-provider")
 
-				component.onChange((value: SearchProvider) => {
-					this.plugin.settings.inlineSearchProvider = value;
-					this.plugin.settingsObservable.setValue(
-						this.plugin.settings
-					);
-					this.plugin.saveSettings();
+			.addText((component) => {
+				component
+					.setValue(this.plugin.settings.inlineSearchProvider.display ?? DEFAULT_SEARCH_PROVIDER.display)
+					.setDisabled(true)
+			})
+			.addExtraButton((component) => {
```

Comment:
Instead of using the extra button, let's use the normal `addButton` and have text that says `Change`.

This makes it really clear without the user having to "guess" or "discover" what the button does. :)

```
.addButton((component) => {
				component.setButtonText("Change");
				component.onClick(() => { //etc
```

![image](https://github.com/andrewmcgivery/obsidian-beautitab/assets/4482878/53851151-5845-4c28-b399-ce07a1b3673d)

Reply:
Oh, good idea!